defos/src/defos_linux.cpp

```diff
@@ -512,6 +519,24 @@ void defos_update()
 
     window_has_focus = focused_window == win;
     apply_cursor_visible();
+
+	// Cursor enter/exit events
+	if (last_cursor_event != cursor_event.type)
```

Comment:
last_cursor_event can happen to be a non-Leave/Enter event. I'd implement this differently. I'd save the entered/left state of the event in a boolean and send the notif if I get an event that is different from the saved state.